src/app/components/ThemeProvider/spacings.ts

```diff
@@ -1,9 +1,16 @@
 import pixelsToRem from '../../utilities/pixelsToRem';
 
+/** 0.5rem (8px) */
```

Comment:
This (and other comments) was to assist with our migration, and documenting the instructions on how to standardise components: https://paper.dropbox.com/doc/Migrate-MostRead-from-Legacy-0FHywUTsf55Brd9VaYUxs